### PR TITLE
Add request, middleware panels

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/Panels/Middlewares/view.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
```
Comment (xepozz) [inline src/Panels/Middlewares/view.html:1]: Is there any opportunity not to use jquery?

File: src/Panels/Request/view.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    </tbody>
+</table>
+
+<script>
+    (function () {
+        let request = collectors[currentDebugSession]["Yiisoft\\Yii\\Debug\\Collector\\RequestCollector"]
+        for (let info in request) {
+            document.querySelector('.panel-request-table tbody').insertAdjacentHTML('beforeend',
+                '<tr><td>' + info + '</td><td>' + request[info] + '</td></tr>');
+        }
```
Comment (xepozz) [inline src/Panels/Request/view.html:19]:
```suggestion
        const request = collectors[currentDebugSession]["Yiisoft\\Yii\\Debug\\Collector\\RequestCollector"]
        const table = document.querySelector('.panel-request-table tbody');
        for (let info in request) {
            table.insertAdjacentHTML('beforeend',
                '<tr><td>' + info + '</td><td>' + request[info] + '</td></tr>');
        }
```

**Review by xepozz — COMMENTED** [review]
👍

File: src/Panels/Middlewares/view.html (hunk 1 above)
Comment (rustamwin) [reply src/Panels/Middlewares/view.html:1]: Not yet. But I'll do it soon without jquery.

**Pull request merged**